File: Clase-DAM-2/Acceso a Datos/Acceso a datos/src/examen/Producto.java
package examen;

public class Producto {

	private String nombre;
	private double precio;

	public void Producto() {
		nombre = "";
		precio = 0;
	}

	public void setNombre(String n) {
		nombre = n;
	}

	public void setPrecio(double p) {
		precio = p;
	}

	public String escribir() {
		return "Nombre: " + nombre + ". Precio: " + precio;
	}

	public void modPrecioPorcentaje(double mod) {
		precio *= mod;
	}
}

File: Santo Domingo Savio/Clase-DAM-2/Multimedia y Dispositivos Moviles/Visual Studio/Papeleria/Papeleria/Producto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Papeleria
{
    class Producto
    {
        public Producto()
        {
            nombre = "";
            tipo = "";
            codigo = 0;
            precio = 0;
        }

        public Producto(string nombre, string tipo, int codigo, double precio)
        {
            this.nombre = nombre;
            this.tipo = tipo;
            this.codigo = codigo;
            this.precio = precio;
        }

        public string nombre { get; }

        public string tipo { get; }

        public int codigo { get; }

        public double precio { get; }

        // Devuelve los datos del producto
        virtual public string Escribir()
        {
            return nombre + ":" + tipo + ":" + codigo + ":" + precio;
        }
    }
}
